### ISSUE TITLE
a problem about traning

### ISSUE THREAD

**Issue by hatsunemiku02** [hwangdonghyun/VNect-TensorFlow-Keras#1]
As the paper said there shoud be two intermidate at res4d and res5a, should I add the whole block of bone length and deltax,y,z and then spit and compute the l2 loss?  Or I just go from res4d and res5a to get the heatmap,ignore the bone length and deltas to compute the l2 loss?
I tried both, it seems if I use the whole block wth bone length and deltas, it will never get 
converged.
Or should I add a loss funtion for the deltas and bone length?
Can you give me any idea about this?
thank u

**Comment by mehtadushy** [comment]
@hatsunemiku02 Just the heatmaps and x,y,z maps are sufficient at res4d and res5a. Also, as a first implementation, ignore all intermediate supervision (res4d, res5a,  and the bone length and delta x,y,z parts of the network)

**Comment by guker** [comment]
> As the paper said there shoud be two intermidate at res4d and res5a, should I add the whole block of bone length and deltax,y,z and then spit and compute the l2 loss? Or I just go from res4d and res5a to get the heatmap,ignore the bone length and deltas to compute the l2 loss?
> I tried both, it seems if I use the whole block wth bone length and deltas, it will never get
> converged.
> Or should I add a loss funtion for the deltas and bone length?
> Can you give me any idea about this?
> thank u

you use bone length as intermediate supervision, how to use it, and what is that performance?

**Comment by guker** [comment]
> @hatsunemiku02 Just the heatmaps and x,y,z maps are sufficient at res4d and res5a. Also, as a first implementation, ignore all intermediate supervision (res4d, res5a, and the bone length and delta x,y,z parts of the network)

@mehtadushy  hello，how much is the weight of heatmap loss，locationmap loss？ 
    how much  is the weight of intermediate supervision loss？thank you very much
